.github/workflows/keep-alive.yml

```diff
@@ -0,0 +1,36 @@
+name: Keep workflows alive
+
+on:
+  # Run on the first of each month at 1:23 UTC
+  schedule:
+  - cron:  '23 1 1 * *'
+  workflow_dispatch:
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Create a minimal payload
+      run: |
+        mkdir _pass
+        touch _pass/pass.txt
```

Comment:
`peaceiris/actions-gh-pages` requires a non-empty payload directory, so this just provides  payload of an empty file.